src/zkvm_verifier/verifier.rs

```diff
@@ -720,16 +735,13 @@ pub fn verify_gkr_circuit<C: Config>(
             .enumerate()
             .for_each(|(idx, pos)| {
                 let val = builder.get(&main_evals, idx);
-                builder.set(
-                    &claims,
-                    *pos,
-                    PointAndEvalVariable {
-                        point: PointVariable {
-                            fs: in_point.clone(),
```

Comment:
@darth-cy We should always avoid the clone of raw stack object. The correct way is to use `builder.eval()` to create a new object.